srcpkgs/python3-halo/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'python3-halo'
+pkgname=python3-halo
+version=0.0.31
+revision=1
+wrksrc="halo-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="
+ python3-log_symbols
+ python3-spinners
+ python3-termcolor
+ python3-colorama
+ python3-six"
```

Comment:
I would put this on one line and wrap it at 80 columns, that's what we do for all other packages.